### PR TITLE
Creates various Raritan models

### PR DESCRIPTION
**Pull request by hooligan-sa**
Creates models for: 
DSX2-16
PX2-5205R
PX3-5292R-M11
PX3-5292R
PX3-5528V-V2
PX3-5729R

### CODE REVIEW

File: device-types/Raritan/DSX2-16-1.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+manufacturer: Raritan
+model: DSX2-16 Console Server
```
Comment (danner26) [inline device-types/Raritan/DSX2-16-1.yaml:3]: The model name appears to be `DSX2-16`
Please remove the "`Console Server`" part

File: device-types/Raritan/PX2-5205R.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+manufacturer: Raritan
+model: PX2-5205R
+slug: px2-5205r
```
Comment (danner26) [inline device-types/Raritan/PX2-5205R.yaml:4]: All slugs should have the manufacturer's name appended to them. This should start with `raritan-`

File: device-types/Raritan/PX3-5292R-M11.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
+---
+manufacturer: Raritan
+model: PX3-5292R-M11
+slug: px3-5292r-m11
```
Comment (danner26) [inline device-types/Raritan/PX3-5292R-M11.yaml:4]: All slugs should have the manufacturer's name appended to them. This should start with `raritan-`

File: device-types/Raritan/PX3-5292R.yaml [hunk 4]
```diff
@@ -0,0 +1,58 @@
+---
+manufacturer: Raritan
+model: PX3-5292R
+slug: px3-5292r
```
Comment (danner26) [inline device-types/Raritan/PX3-5292R.yaml:4]: All slugs should have the manufacturer's name appended to them. This should start with `raritan-`

File: device-types/Raritan/PX3-5729R.yaml [hunk 5]
```diff
@@ -0,0 +1,46 @@
+---
+manufacturer: Raritan
+model: PX3-5729R
+slug: px3-5729r
+part_number: PX3-5729R
+u_height: 1
+is_full_depth: false
```
Comment (danner26) [inline device-types/Raritan/PX3-5729R.yaml:7]: Please add the `airflow` data point if you have/can find it

File: device-types/Raritan/PX3-5292R.yaml [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+part_number: PX3-5292R
+u_height: 1
+is_full_depth: false
```
Comment (danner26) [inline device-types/Raritan/PX3-5292R.yaml:7]: Please add the `airflow` data point if you have/can find it

File: device-types/Raritan/PX3-5292R-M11.yaml [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+part_number: PX3-5292R-M11
+u_height: 1
+is_full_depth: false
```
Comment (danner26) [inline device-types/Raritan/PX3-5292R-M11.yaml:7]: Please add the `airflow` data point if you have/can find it

File: device-types/Raritan/PX2-5205R.yaml [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+part_number: PX2-5205R
+u_height: 0
+is_full_depth: false
```
Comment (danner26) [inline device-types/Raritan/PX2-5205R.yaml:7]: Please add the `airflow` data point if you have/can find it

File: device-types/Raritan/DSX2-16-1.yaml [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+slug: dsx2-16-1
+part_number: DSX2-16
+comments: Dominion SX II Console Server
+u_height: 1
+is_full_depth: false
```
Comment (danner26) [inline device-types/Raritan/DSX2-16-1.yaml:8]: Please add the `airflow` data point if you have/can find it

Comment (danner26) [inline device-types/Raritan/DSX2-16-1.yaml:8]: Please add the `weight` and `weight_unit` data point if you have/can find it

File: device-types/Raritan/PX2-5205R.yaml (hunk 8 above)
Comment (danner26) [inline device-types/Raritan/PX2-5205R.yaml:7]: Please add the `weight` and `weight_unit` data point if you have/can find it

File: device-types/Raritan/PX3-5292R-M11.yaml (hunk 7 above)
Comment (danner26) [inline device-types/Raritan/PX3-5292R-M11.yaml:7]: Please add the `weight` and `weight_unit` data point if you have/can find it

File: device-types/Raritan/PX3-5292R.yaml (hunk 6 above)
Comment (danner26) [inline device-types/Raritan/PX3-5292R.yaml:7]: Please add the `weight` and `weight_unit` data point if you have/can find it

File: device-types/Raritan/PX3-5528V-V2.yaml [hunk 10]
```diff
@@ -0,0 +1,94 @@
+---
+manufacturer: Raritan
+model: PX3-5528V-V2
+slug: px3-5528v-v2
+part_number: PX3-5528V-V2
+u_height: 0
+is_full_depth: false
```
Comment (danner26) [inline device-types/Raritan/PX3-5528V-V2.yaml:7]: Please add the `weight` and `weight_unit` data point if you have/can find it

File: device-types/Raritan/PX3-5729R.yaml (hunk 5 above)
Comment (danner26) [inline device-types/Raritan/PX3-5729R.yaml:7]: Please add the `weight` and `weight_unit` data point if you have/can find it

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/Raritan/DSX2-16-1.yaml (hunk 1 above)
Comment (hooligan-sa) [reply device-types/Raritan/DSX2-16-1.yaml]: Removed

File: device-types/Raritan/DSX2-16-1.yaml [hunk 11]
```diff
@@ -0,0 +1,56 @@
+---
+manufacturer: Raritan
+model: DSX2-16 Console Server
+slug: dsx2-16-1
```
[the remark this answers is not in the document]
Comment (hooligan-sa) [reply device-types/Raritan/DSX2-16-1.yaml]: Manufacturer name prepended

File: device-types/Raritan/PX3-5292R-M11.yaml (hunk 3 above)
Comment (hooligan-sa) [reply device-types/Raritan/PX3-5292R-M11.yaml]: Manufacturer name prepended

File: device-types/Raritan/PX3-5292R.yaml (hunk 4 above)
Comment (hooligan-sa) [reply device-types/Raritan/PX3-5292R.yaml]: Manufacturer name prepended

File: device-types/Raritan/PX3-5292R-M11.yaml (hunk 7 above)
Comment (hooligan-sa) [reply device-types/Raritan/PX3-5292R-M11.yaml:7]: weight and weight_unit added

File: device-types/Raritan/PX3-5528V-V2.yaml (hunk 10 above)
Comment (hooligan-sa) [reply device-types/Raritan/PX3-5528V-V2.yaml:7]: weight and weight_unit added

File: device-types/Raritan/DSX2-16-1.yaml [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+part_number: DSX2-16
+comments: Dominion SX II Console Server
```
[the remark this answers is not in the document]
Comment (hooligan-sa) [reply device-types/Raritan/DSX2-16-1.yaml:6]: Added

File: device-types/Raritan/DSX2-16-1.yaml (hunk 9 above)
Comment (hooligan-sa) [reply device-types/Raritan/DSX2-16-1.yaml:8]: Was unable to find it, sorry

File: device-types/Raritan/PX2-5205R.yaml (hunk 8 above)
Comment (hooligan-sa) [reply device-types/Raritan/PX2-5205R.yaml:7]: It's a PDU, no fans/no airflow that I'm aware of

File: device-types/Raritan/PX3-5292R-M11.yaml [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+comments: Rack PDU, Metered, Remote Switching, 7.4kVA, 230V, (12) C13
```
[the remark this answers is not in the document]
Comment (hooligan-sa) [reply device-types/Raritan/PX3-5292R-M11.yaml:10]: Added

File: device-types/Raritan/PX3-5292R.yaml [hunk 14]
```diff
@@ -0,0 +1,58 @@
...
+comments: Rack PDU, Metered, Remote Switching, 7.4kVA, 230V, (12) C13
```
[the remark this answers is not in the document]
Comment (hooligan-sa) [reply device-types/Raritan/PX3-5292R.yaml:10]: Added

File: device-types/Raritan/PX3-5729R.yaml (hunk 5 above)
Comment (hooligan-sa) [reply device-types/Raritan/PX3-5729R.yaml:7]: It's a PDU, so no fans/airflow information that I'm aware of

File: device-types/Raritan/PX3-5729R.yaml [hunk 15]
```diff
@@ -0,0 +1,46 @@
...
+comments: Rack PDU, Metered, Remote Switching, 7.4kVA, 230V, (4) C13, (4) C19
```
[the remark this answers is not in the document]
Comment (hooligan-sa) [reply device-types/Raritan/PX3-5729R.yaml:10]: Added

File: device-types/Raritan/PX3-5528V-V2.yaml [hunk 16]
```diff
@@ -0,0 +1,94 @@
+---
+manufacturer: Raritan
+model: PX3-5528V-V2
+slug: px3-5528v-v2
```
[the remark this answers is not in the document]
Comment (hooligan-sa) [reply device-types/Raritan/PX3-5528V-V2.yaml]: prepended with manufacturer name

File: device-types/Raritan/PX3-5729R.yaml [hunk 17]
```diff
@@ -0,0 +1,46 @@
+---
+manufacturer: Raritan
+model: PX3-5729R
+slug: px3-5729r
```
[the remark this answers is not in the document]
Comment (hooligan-sa) [reply device-types/Raritan/PX3-5729R.yaml]: manufacturer prepended

**Comment by hooligan-sa** [comment]
Not sure why this has gone stale, can @danner26 review/advise if there's still something awaiting review?

**Review by hooligan-sa — COMMENTED** [review]
All changes completed that I'm aware of, please advise if that's not the case

**Review by danner26 — CHANGES_REQUESTED** [review]
Please revert the file permission on `PX3-5497V.yaml` back to `0755`

**Comment by danner26** [comment]
Hello, we have released a new set of tests and features to assist with properly creating definition files for the DeviceType-Library. These tests will ensure that all files conform to a standardized format and contain valid data. Over the coming weeks we will be adding more tests to handle other currently undefined cases. If you notice anything that you believe is a bug, please @ me in a comment and details what you think is incorrect.

Along with additional test cases we have added a feature that will annotate why tests fail. To access these annotations after a test has completed, please navigate to the _Files changed_ tab within your PR. At the very bottom of the page you will see a list of errors along with the filename in question and the error which was found.

With all this being said, if your PR is validated then great! Please @ me me so I can review it further! If your PR is not valid, please review the issues and correct them; in the meantime I will mark your PR as a draft. After correcting the issues and recieving a valid status, please mark the PR as ready to be reviewed. Thank you!

File: device-types/Raritan/DSX2-16-1.yaml (hunk 1 above)
Comment (hooligan-sa) [reply device-types/Raritan/DSX2-16-1.yaml]: Remove "-1" and committed file "DSX2-16.yaml"

**Comment by hooligan-sa** [comment]
I've removed the DSX2-16-1.yaml file and replaced it with DSX2-16.yaml
In that process I've updated the model, slug and part_number fields, and all tests on this file pass now - Please shout if there are still any issues?

**Comment by hooligan-sa** [comment]
Hi Daniel,

I really can’t understand the requirements for those permissions, but I’ve reset them.

Regards,
Patrick Holahan

From: "Daniel W. Anner" ***@***.***>
Reply to: netbox-community/devicetype-library ***@***.***>
Date: Tuesday, 18 July 2023 at 15:13
To: netbox-community/devicetype-library ***@***.***>
Cc: Patrick Holahan ***@***.***>, Author ***@***.***>
Subject: Re: [netbox-community/devicetype-library] Creates various Raritan models (PR #1386)


@danner26 requested changes on this pull request.

________________________________

On device-types/Raritan/PX3-5497V.yaml<https://github.com/netbox-community/devicetype-library/pull/1386#discussion_r1266755910>:

Please revert the permission change you made here. If you need help with this, you are welcome to give me access to your fork and I can make the adjustment for you

**Review by danner26 — APPROVED** [review]

**Pull request merged**